SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-intro.md

```diff
@@ -19,21 +19,33 @@ Following this tutorial, you implement cross-server remote procedure calls (RPCs
 _Image: Example of cross-server shooting using multiserver zoning functionality._ </br></br>
 
 The tutorial demonstrates that the workflows and iteration speed you’re used to as an Unreal developer are almost entirely unaltered by the GDK: it’s just like regular Unreal.
-</br>
-</br>
-</br>
-**Notes:**
 
-* Before starting this tutorial you need to complete the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). 
+### Change your branch of the Example Project 
+
+Before starting this tutorial you need to complete the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). Following this, you clone or download and set up all the elements of the GDK and its dependencies, including the Example Project.
```

Comment:
```suggestion
Before starting this tutorial you need to complete the [Example Project setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). Following this, you clone or download and set up all the elements of the GDK and its dependencies, including the Example Project.
```
Couple of things:
- set up --> setup
- does "Following this" mean "after this", or "while you follow this"?